classical-components-demo/licenseplate-scanner/src/main/java/io/scanbot/example/ScannerActivity.kt

```diff
@@ -0,0 +1,83 @@
+package io.scanbot.example
+
+import android.os.Bundle
+import android.util.TypedValue
+import android.widget.Button
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import io.scanbot.sdk.ScanbotSDK
+import io.scanbot.sdk.SdkLicenseError
+import io.scanbot.sdk.camera.CameraOpenCallback
+import io.scanbot.sdk.camera.CameraPreviewMode
+import io.scanbot.sdk.camera.FrameHandlerResult
+import io.scanbot.sdk.licenseplate.LicensePlateScanResult
+import io.scanbot.sdk.licenseplate.LicensePlateScanStrategy
+import io.scanbot.sdk.licenseplate.LicensePlateScannerFrameHandler
+import io.scanbot.sdk.ui.camera.*
+
+class ScannerActivity : AppCompatActivity() {
+    private val scanbotSdk = ScanbotSDK(this)
+    private val licensePlateScanner = scanbotSdk.licensePlateScanner()
+
+    private lateinit var cameraView: IScanbotCameraView
+    private lateinit var resultTextView: TextView
+
+    private var useFlash = false
+
+    private lateinit var licensePlateScannerFrameHandler: LicensePlateScannerFrameHandler
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_scanner)
+        licensePlateScanner.scanStrategy = LicensePlateScanStrategy.LicensePlateML
+
+        cameraView = findViewById<ScanbotCameraXView>(R.id.cameraView)
```

Comment:
Some parts of CameraX are still in alpha so we should not encourage SDK users to use this component, aren't we?